CleanSweep_Group6/src/main/java/edu/depaul/cleanSweep/controlSystem/Cleaner.java

```diff
@@ -218,4 +231,14 @@ public String printCoordinate() {
 	public void changeHeading(char h){
 		headingTowards = h;
 	}
+
+	public char getHeading() {return this.headingTowards;}
+
+	public ArrayList<FloorTile> getCleanerHistory(){
+		return this.cleanerHistory;
+	}
+
+	private FloorTile copyTile(FloorTile tileToCopy){
+		return new FloorTile(tileToCopy._y, tileToCopy._x);
```

Comment:
Need to also copy the links from each node or the cleaner won't remember how to return